mock-data/CosmosDbSeeder/items/index_data/site_ABC02.json

```diff
@@ -9,5 +9,43 @@
       -1.66382134,
       53.79628754
     ]
-  }
+  },
+  "attributeValues": [
```

Comment:
This is literally the most nittiest of nitpicks so feel free to absolutely tell me where to go, but with the user documents we go for just "roles" rather than "roleValues", and I wonder if this likewise should be just "attributes"? 

(but like I say please feel free to completely ignore me!)